README.md

```diff
@@ -1 +1,38 @@
-# serverless-api
\ No newline at end of file
+# serverless-api
+
+## Project: serverless api on AWS
+
+### Author: Matt Ravenmoore
+
+### Links and Resources
+
+- [ci/cd](https://github.com/ravenmoore-401-JS/serverless-api/actions) (GitHub Actions)
+- [back-end server url](https://v5ar4xfvsf.execute-api.us-west-2.amazonaws.com/dragon)
+
+- [submit PR](https://github.com/ravenmoore-401-JS/serverless-api/pull/3)
+
+### Setup
+
+clone down the project from github
+run `npm i` in each folder for lambdas
+zip each lambda folder and patch to aws
+
+
+#### `.env` requirements (where applicable)
+
+
+#### How to initialize/run your application (where applicable)
+
+
+#### How to use your library (where applicable)
+
+#### Tests
+
+- How do you run tests?
+to check tests run `npm test`
+
+- Any tests of note?
+a few basic status tests for server.
+
+#### UML
+soon to come
```

Comment:
I really need to see all that documentation added here that were instructions:
- What is the root URL to your API?
- What are the routes?
- What inputs do they require?
- What output do they return?
And I would also like to see tests attempted